reana-serial.yaml

```diff
@@ -0,0 +1,31 @@
+version: 0.1.0
+metadata:
+  authors:
+   - Ana Trisovic <ana.trisovic@gmail.com>
+   - Lukas Heinrich <lukas.heinrich@gmail.com>
+   - Tibor Simko <tibor.simko@cern.ch>
+  title: ROOT6 and RooFit physics analysis example
+  date: 19 February 2018
+  repository: https://github.com/reanahub/reana-demo-root6-roofit/
+code:
+  files:
+   - code/gendata.C
+   - code/fitdata.C
+inputs:
+  parameters:
+    events: 20000
+    gendata: code/gendata.C
+    fitdata: code/fitdata.C
+outputs:
+  files:
+   - outputs/plot.png
+environments:
+  - type: docker
+    image: reanahub/reana-env-root6
+workflow:
+  type: serial
+  specification:
```

Comment:
The `specification` looks a bit redundant... Perhaps it's enough to have `steps` directly?  Dunno if people will be writing Yadage etc right inside `reana.yaml` -- it would be probably too big.